tests/release/service/tenant_pipelines.go

```diff
@@ -3,68 +3,80 @@ package service
 import (
 	"encoding/json"
 	"fmt"
+	kubeapi "github.com/konflux-ci/e2e-tests/pkg/clients/kubernetes"
 	"time"
 
 	tektonutils "github.com/konflux-ci/release-service/tekton/utils"
 	"k8s.io/apimachinery/pkg/runtime"
 
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
 	appservice "github.com/konflux-ci/application-api/api/v1alpha1"
 	"github.com/konflux-ci/e2e-tests/pkg/framework"
 	"github.com/konflux-ci/e2e-tests/pkg/utils"
-	"k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	releasecommon "github.com/konflux-ci/e2e-tests/tests/release"
 	releaseApi "github.com/konflux-ci/release-service/api/v1alpha1"
-	corev1 "k8s.io/api/core/v1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
 	tektonv1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 var _ = framework.ReleaseServiceSuiteDescribe("Release service tenant pipeline", Label("release-service", "tenant"), func() {
 	defer GinkgoRecover()
 
 	var fw *framework.Framework
 	AfterEach(framework.ReportFailure(&fw))
+	var kubeAdminClient *framework.ControllerHub
 	var err error
-	var tenantNamespace string
+	var devNamespace = "tenant-dev"
 	var releasedImagePushRepo = "quay.io/redhat-appstudio-qe/dcmetromap"
 	var sampleImage = "quay.io/redhat-appstudio-qe/dcmetromap@sha256:544259be8bcd9e6a2066224b805d854d863064c9b64fa3a87bfcd03f5b0f28e6"
 	var gitSourceURL = "https://github.com/redhat-appstudio-qe/dc-metro-map-release"
 	var gitSourceRevision = "d49914874789147eb2de9bb6a12cd5d150bfff92"
 	var tenantServiceAccountName = "tenant-service-account"
-	var tenantPullSecretName="tenant-pull-secret"
+	var tenantPullSecretName = "tenant-pull-secret"
 
 	var releaseCR *releaseApi.Release
 	var snapshotPush *appservice.Snapshot
+	var testEnvironment = utils.GetEnv("TEST_ENVIRONMENT", releasecommon.UpstreamTestEnvironment)
 
 	BeforeAll(func() {
-		// Initialize the tests controllers
-		fw, err = framework.NewFramework(utils.GetGeneratedNamespace("tenant-dev"))
-		Expect(err).NotTo(HaveOccurred())
-		tenantNamespace = fw.UserNamespace
+		if testEnvironment == releasecommon.DownstreamTestEnvironment {
+			fw, err = framework.NewFramework(utils.GetGeneratedNamespace(devNamespace))
+			Expect(err).NotTo(HaveOccurred())
+			devNamespace = fw.UserNamespace
```

Comment:
Below sentence need to be added in this block?
`kubeAdminClient = fw.AsKubeAdmin`